releasenotes/notes/jmxfetch-0.23.0-concurrency-support-14de5c08f927bd1c.yaml

```diff
@@ -0,0 +1,19 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    JMXFetch 0.23.0 introduces support for concurrent instance
+    metric collection. Concurrent collection should avoid stalling
+    healthy instances in the event of issues (networking, system) in 
+    any of the other instances configured. Threenew settings are
```

Comment:
typo: `Three new`

Reply:
The detailed description of these settings is easy to understand IMO 👌

I think we should leave it out of the release note though, and rather document this in a more "permanent" location (datadog.yaml for example)